File: app/src/main/java/banbara23/com/kotlinjacksonsample/ApiService.java
package banbara23.com.kotlinjacksonsample;

import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.POST;

public interface ApiService {
    @POST("/")
    Observable<PostResponse> post();

    @GET("/")
    Observable<GetResponse> get();
}

File: app/src/main/java/banbara23/com/kotlinjacksonsample/ApiService.kt
package banbara23.com.kotlinjacksonsample

import io.reactivex.Observable
import retrofit2.http.GET
import retrofit2.http.POST

interface ApiService {
    @POST("/")
    fun post(): Observable<PostResponse>

    @GET("/")
    fun get(): Observable<GetResponseJ>
}